src/aac/template_engine.py

```diff
@@ -18,9 +18,9 @@ def load_templates(package_name: str, template_directory: str = "templates") ->
     Returns:
         list of loaded templates
     """
-
```

Comment:
I think for inner functions we want to keep the space here.

Reply:
Done